plugin-compass/src/main/java/com/mapbox/maps/plugin/compass/CompassViewImpl.kt

```diff
@@ -52,7 +52,7 @@ open class CompassViewImpl
       visibility = if (value) {
         View.VISIBLE
       } else {
-        View.INVISIBLE
+        View.GONE
```

Comment:
should we cover such cases with tests?

Reply:
Do you mean pixel match test?

Reply:
As discussed online, will leave it as it is now since it's not easy to test.